config/environments/test.rb

```diff
@@ -39,4 +39,5 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
```suggestion
  WebMock.disable_net_connect!(allow_localhost: false)
```

Alguna razón para que lo quisieras permitir? Idealmente en test nunca se debería hacer una petición real